### PR TITLE
Update To Include CORD-19 Bounty Mirror

### PR DESCRIPTION
**Pull request by bonedaddy**
For https://github.com/ipfs/ipfs-cluster/issues/1021, not done yet still want to add docker tooling, mostly just looking for PR feedback incase something needs to be changed.

### CODE REVIEW

**Review by hsanjuan — COMMENTED** [review]
LGTM. If you want to prettify the join url with dnslink we can do it too.

File: index.md [hunk 1]
```diff
@@ -54,6 +54,16 @@
 
 ---
 
+| | [CORD-19 Mirror](https://pages.semanticscholar.org/coronavirus-research) |
+| - | - |
+| **Description** | COVID-19 research dataset mirror known as CORD-19 |
```
Comment (jaller94) [inline index.md]:
```suggestion
| **Description** | COVID-19 research datasets mirror known as CORD-19 |
```

File: index.md [hunk 2]
```diff
@@ -54,6 +54,16 @@
...
+| ***Init&Run*** | `ipfs-cluster-follow cord19-datase init http://localhost:8080/ipfs/QmdhKHCaojZDquDok7uNnEuPFbzHbsGMnq1g3kdBJc8MPh` |
```
Comment (jaller94) [inline index.md]:
```suggestion
| ***Init&Run*** | `ipfs-cluster-follow cord19-datasets init http://localhost:8080/ipfs/QmdhKHCaojZDquDok7uNnEuPFbzHbsGMnq1g3kdBJc8MPh` |
```

**Comment by bonedaddy** [comment]
@hsanjuan good idea, looks a lot nicer i've made the change
@jaller94 thanks for pointing out the typos, i've commited your suggestions (side note: when batch commit :cry: )

**Comment by RubenKelevra** [comment]
Can you change the call to `run --init` instead of just `init`? To get in line with the other cluster commands :)

**Comment by bonedaddy** [comment]
> Can you change the call to `run --init` instead of just `init`? To get in line with the other cluster commands :)

That seems like an overly excessive nitpick  to be honest. There is no difference at all.

edit:

Typos, and DNSLink instead of a very long HTTP based path make sense and is entirely fair. Changing `init` to `run --init` seems like  a nitpick when there doesn't need to be one. That being said if a pr change request Is made I'll do it.

**Comment by RubenKelevra** [comment]
> That seems like an overly excessive nitpick to be honest. There is no difference at all.

Well, just `init` won't run the cluster follower and just stop after the initialization. The goal is to provide one command to start following and don't have the user to figure out, why it's stopped after a second. :)

**Comment by bonedaddy** [comment]
> > That seems like an overly excessive nitpick to be honest. There is no difference at all.
> 
> Well, just `init` won't run the cluster follower and just stop after the initialization. The goal is to provide one command to start following and don't have the user to figure out, why it's stopped after a second. :)

Well the output from the `init` command quite clearly says "run `insert comand here` to start following the cluster". I don't think its fair to say that every time you run init, you need to immediately start following the cluster afterwards. 

There could be any number of reasons why you want to wait after running `init`, and the `init` command as the ~very final line~ 2nd to last line of output, literally says "run this command to start following".

That being said as I mentioned in the last comment, if there is an official change request made I will do it.

**Comment by parkan** [comment]
hi @bonedaddy, when you're done, could you squash your commits and add "fixes ipfs/ipfs-cluster#1021" (as per https://github.blog/2013-03-18-closing-issues-across-repositories/) to the final commit message?

this is just for our bounty automations

**Comment by bonedaddy** [comment]
> hi @bonedaddy, when you're done, could you squash your commits and add "fixes [ipfs/ipfs-cluster#1021](https://github.com/ipfs/ipfs-cluster/issues/1021)" (as per https://github.blog/2013-03-18-closing-issues-across-repositories/) to the final commit message?
> 
> this is just for our bounty automations

sure, I'll do that now and resquash if changes need to be made

edit: done, and yikes that was more difficult than I anticipated, probably need to brush up on my squashing ability.

File: index.md [hunk 3]
```diff
@@ -54,6 +54,14 @@
 
 ---
 
+| | [CORD-19 Mirror](https://pages.semanticscholar.org/coronavirus-research) |
+| - | - | 
+| **Description** | COVID-19 research datasets mirror known as CORD-19 |
+| ***Init&Run*** | `ipfs-cluster-follow cord19-datasets init cord19.temporal.cloud` |
```
Comment (hsanjuan) [inline index.md]:
```suggestion
| ***Init&Run*** | `ipfs-cluster-follow cord19-datasets run --init cord19.temporal.cloud` |
```

**Review by hsanjuan — DISMISSED** [review]
Since the row title is `Init&Run`, I guess it should init and run. A nitpick but cheap to fix.

File: index.md (hunk 3 above)
Comment (hsanjuan) [reply index.md]: I can't commit this directly :/

**Comment by hsanjuan** [comment]
Cluster works for me.

**Comment by bonedaddy** [comment]
Ok it should be good to go, I committed the suggestion, and squashed to include the commit msg that @parkan asked for.

**Review by hsanjuan — APPROVED** [review]

**Comment by hsanjuan** [comment]
Thanks! :1st_place_medal:

**Pull request merged**